### ISSUE TITLE
Static Site to display all the formulas as code snippets in different languages?

### ISSUE THREAD

**Issue by mdew192837** [BaReinhard/Hacktoberfest-Mathematics#16]

**Comment by BaReinhard** [comment]
Any thoughts on how this should be structured?

**Comment by mdew192837** [comment]
Category (Physics, Math) then by common formulas then by language

**Comment by BaReinhard** [comment]
Good call, this seems more intuitive, I believe I'll create a contribution file to a similar structure as this, it would make a site easier to model as well.

**Comment by BaReinhard** [comment]
I'll work on looking into the logic for this, should be easy to create a gh-pages repo to handle this.

**Comment by BaReinhard** [comment]
Should have a working prototype by midday tomorrow

**Comment by Adroit11** [comment]
I would love to give a helping hand on the static page UI.

**Comment by BaReinhard** [comment]
I decided a static page isn't quite as fun, I used react to create a basic prototype and it uses the github api to render the content, take a look here:
https://bareinhard.github.io/Hacktoberfest-Mathematics/

it is very rough, but the functionality is almost there.

**Comment by Adroit11** [comment]
Yeap the functionality is their, did like what you came up with. So are you still looking for contributions on the page?
If yes make the code available and less roll in the pull request for modifications.

**Comment by BaReinhard** [comment]
I made a branch called webpage-build , it was a pretty rough prototype so things aren't named very well at all and it's a bit of a mess with no comments. I don't believe it pushed the latest changes yet but I'll be sure to push them and make comments tomorrow.

**Comment by BaReinhard** [comment]
@mdew192837 any suggestions on what else to include on the web page. I'm thinking of using an autocomplete input box to more easily see the available formulas.
